src/Modal/Modal.component.tsx

```diff
@@ -31,10 +32,12 @@ const StyledModal = StyledReactModal.default.styled`
     flex-direction: column;
     justify-content: space-between;
 
-    border-radius: modal;
-    ${({ background = 'neutrals.white.base' }: ModalProps) =>
-        css({ backgroundColor: background })};
-    ${({ color }: ModalProps) => css({ color })}
+    ${({ background = 'neutrals.white.base', color }: ModalProps) =>
+        css({
+            background: th.color(background),
```

Comment:
`linear-gradient` didn't work for `backgroundColor`, so I switched it back to `background` but manually run it through the theme color.